Neos.Flow/Classes/ResourceManagement/ResourceManager.php

```diff
@@ -260,6 +260,40 @@ public function importUploadedResource(array $uploadInfo, $collectionName = self
         return $resource;
     }
 
+    /**
+     * Import a deferred resource into it's destination collection and persist it.
```

Comment:
```suggestion
     * Import a deferred resource into it's destination collection and persist it.
     * This will do nothing, if the resource is not deferred (has no source attached).
```